cliquet/schema.py

```diff
@@ -61,6 +61,12 @@ def is_readonly(self, field):
         """
         return field in self.get_option("readonly_fields")
 
+    @classproperty
+    def missing(cls):
+        if cls != ResourceSchema:
```

Comment:
Actually I would remove this sanity check.
If there is a schema, the missing properties in the schema will take care of raising an error (see /buckets/id/groups/id for instance) We cannot assume that only ResourceSchema accepts empty data.